cargo-dist/src/tasks.rs

```diff
@@ -1153,6 +1158,10 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             .install_path
             .clone()
             .unwrap_or(vec![InstallPathStrategy::CargoHome]);
+        let install_success_msg = package_config
+            .install_success_msg
+            .clone()
+            .unwrap_or("everything's installed!".to_string());
```

Comment:
```suggestion
        let install_success_msg = package_config
            .install_success_msg
            .as_deref()
            .unwrap_or("everything's installed!")
            .to_owned();
```

clippy will probably complain about the unconditional to_string